src/amo/img/icons/fox-puzzle.svg

```diff
@@ -13,5 +13,5 @@
   <path
      id="Shape-Copy-2"
      d="M 104.32734,54.777526 C 98.627711,48.447448 92.928083,42.117369 87.228454,35.787291 83.445981,32.257136 85.569945,24.440494 91.496073,26.826209 98.996948,28.989225 106.14963,20.279304 103.19751,13.25341 100.67334,7.2366168 95.641986,1.2639199 88.938391,0.05471855 82.550814,-0.56123288 76.356487,5.6476755 77.564303,12.110981 c 0.885165,6.100691 -7.15136,5.860444 -9.807164,2.051223 C 64.704339,11.124663 62.317029,7.2921574 58.6836,4.9382175 54.900999,2.9824181 51.061474,5.426393 48.508292,8.1748523 44.562389,11.72776 40.616486,15.280667 36.670583,18.833574 32.773101,22.237796 25.635486,19.532321 27.63815,13.814852 29.319644,6.6454277 21.666173,-0.65219384 14.632091,2.1240512 8.4452351,4.577096 2.5833353,9.552239 0.61065759,16.077813 -0.85716325,22.744267 5.7737324,28.920685 12.263246,28.424615 c 5.653451,-0.477237 6.332291,6.845643 2.806938,9.858046 -4.524497,4.192925 -9.3068141,8.133716 -13.6584477,12.495819 -2.684923,3.397386 -1.21012403,8.12913 1.856331,10.70242 4.0252072,4.129415 7.3548877,9.046133 12.1077037,12.365686 5.16487,3.715321 12.418635,-1.473229 10.922986,-7.53462 -0.855938,-5.428953 4.675701,-11.2193 10.0956,-8.485202 5.363514,2.613337 10.670659,8.676765 9.157464,15.017158 -2.332101,6.248881 -9.74175,3.638763 -14.640181,4.633785 -5.868481,2.530811 -4.853248,10.546106 -0.540054,14.104696 5.327219,5.776472 10.389854,11.823747 15.888298,17.425047 3.412592,2.67384 8.133056,1.15466 10.709554,-1.90244 6.104111,-5.65883 12.565406,-10.963506 18.408192,-16.881291 4.744603,-4.157531 2.465713,-13.22138 -4.33048,-13.029977 -5.429022,0.604905 -12.520366,-2.955011 -11.103228,-9.332515 1.459373,-7.243044 9.77701,-13.70986 17.173415,-11.002401 6.115027,2.979012 2.773054,10.591112 5.151349,15.471702 3.346713,5.444474 10.88941,3.15082 14.374191,-0.947644 2.914892,-2.926735 6.591823,-5.215394 8.831903,-8.716587 1.22576,-2.553324 0.78013,-5.799337 -1.14744,-7.888771 z"
-     style="fill:#ffffff;fill-opacity:1;fill-rule:evenodd;stroke:none" />
+     style="fill:#ffb200;fill-opacity:1;fill-rule:evenodd;stroke:none" />
```

Comment:
I figured another loader would be the solution but thought I might address it outside this PR.

I'll file an issue and tackle it separately 👍